src/txn/txn_rollback_to_stable.c

```diff
@@ -967,17 +967,9 @@ __rollback_to_stable_btree(WT_SESSION_IMPL *session, wt_timestamp_t rollback_tim
 static int
```

Comment:
During recovery, currently when the stable timestamp is not set, we never perform RTS. It will give problem to the standalone releases where no timestamps are enabled. You need to remove that check there.